src/content/developers/docs/standards/index.md

```diff
@@ -23,9 +23,20 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
 
 ## Types of standards {#types-of-standards}
 
-Certain EIPs relate to application-level standards (e.g. a standard smart-contract format), which are introduced as [Ethereum Requests for Comment (ERC)](https://eips.ethereum.org/erc). Many ERCs are critical standards used widely across the Ethereum ecosystem.
+There are 3 types of EIPs:
 
-- [List of ERCs](https://eips.ethereum.org/erc)
+- [Standards Track](https://eips.ethereum.org/): describes any change that affects most or all Ethereum implementations
+- [Meta Track](https://eips.ethereum.org/meta): describes a process surrounding Ethereum or proposes a change to a process
+- [Informational Track](https://eips.ethereum.org/informational): describes an Ethereum design issue or provides general guidelines or information to the Ethereum community
```

Comment:
Same as comment above. IMO, "Informational EIPs" will be better.

Reply:
Likewise, this is the correct term to use here.